lib/DependencyProvider.js

```diff
@@ -258,7 +254,7 @@ function onlineVersionsFromScratch(
 // Helper functions ##################################################
 
 function semverCompare(a /*: string */, b /*: string */) /*: number */ {
-  return collator.compare(a, b);
+  return -collator.compare(a, b);
```

Comment:
This is also possible. I'm fine with either implementation, but I feel like using `-` makes it slightly more obvious that we're sorting in the opposite order.

```suggestion
  return -collator.compare(b, a);
```

Reply:
If this function (`semverCompare`) was written in Elm, we would probably have returned an `Order`. Following [Basics.compare](https://elm.dmy.fr/packages/elm/core/latest/Basics#compare), I would expect:

- `LT` if `a` is less than `b` (semver wise)
- `EQ` if they are the same
- `GT` if `a` is greater than `b`

In JavaScript, my understanding is:

- LT = -1
- EQ = 0
- GT = 1

So now this function returns the opposite of what one would expect (because that’s what we need where it’s used now)? Should we rename in to `flippedSemverCompare`? If so, is `collator.compare(b, a)` better? 🤔

Reply:
I'm fine with that. I changed it to what you suggested and added a small explanation.